app/components/primer/alpha/banner.rb

```diff
@@ -3,6 +3,13 @@
 module Primer
   module Alpha
     # Use `Banner` to highlight important information.
+    #
+    # @accessibility
+    #   Depending on the scenario, some Banners may need to receive focus when they appear. This helps to maximize discoverability of the message, especially in critical scenarios. Defer to the accessibility team to determine if your scenario requires focusing the banner.
```

Comment:
I think it would be helpful to expand on the scenarios (when to focus?, when to polite live region?) BUT I can see this easily getting really long given all the nuances. This is probably something we'd want to add in the "Overview" section as part of the parallel coaching engagement.

For the issue, a clear recommendation on how one should focus with a Rails Banner is sufficient 👍 . If the API is updated to support this (based on https://github.com/primer/view_components/pull/2576, for example), we can then update this!

Reply:
Good call! I added this info to the [Overview section of the main page](https://github.com/primer/design/pull/722) and I'm cross-linking to it. What do you think about this?

Reply:
This looks good! Sorry my wording was super confusing - I didn't expect you to add the overview section since we will look at it as part of the coaching work but thanks for doing it anyways haha! Looks good to me and Chelsea and I can build on what you have!